boards/xtensa/intel_s1000_crb/doc/index.rst

```diff
@@ -125,6 +125,12 @@ software keys you have downloaded from `Tensilica Tools for Sue Creek`_
    follow what the license manager expects or create a dedicated ethernet device
    for this to work.
 
+   On the latest LTS ubuntu(18.04), you can install the license key manually with
```

Comment:
Ubuntu needs an uppercase first letter and I'd remove the word "latest" (since 19 is available too):  ``On Ubuntu 18.04 LTS``.

Reply:
Hi David, I addressed with commit 2b77c6b.